test/AsmJs/uint.js

```diff
@@ -12,10 +12,9 @@ let m = function (stdlib, foreign, heap) {
   }
   return f;
 }({}, {}, asmHeap);
-print(m(4294967295));
-print(m(4294967295));
 
-m = function (stdlib, foreign, heap) {
+
+m2 = function (stdlib, foreign, heap) {
```

Comment:
yup, it is because of defer parsing. depending whether we defer or not the message will be in different place (this is why so many of the asm.js tests have some -off:deferparse or -forcedeferparse flag)